static/js/appjs/google_analytics.js

```diff
@@ -1,7 +1,7 @@
 (function() {
     'use strict';
 
-    var sendToGA = function(eventCategory, eventLabel, eventAction='click') {
+    var sendToGA = function(eventCategory, eventLabel, eventAction) {
```

Comment:
😄 it's not that simple.
Analyze the usage of sendToGA in the file and see where click was used as a default parameter and update the function calls accordingly

Reply:
Made all the changes except this one, I will look into it. Until then let me know if there are other changes to be done for this issue :dancer:

Reply:
Made these changes